examples/3dtiles_loader.html

```diff
@@ -90,7 +90,8 @@
             const view = new GlobeView(viewerDiv, placement, {
                 controls: {
                     minDistance: 100,
-                }
+                },
+		activateSunlight: false,
             });
```

Comment:
Tabulations instead of spaces

```suggestion
                },
                activateSunlight: false,
            });
```